### ISSUE TITLE
Format open / all files when changing settings

### ISSUE THREAD

**Issue by paul-dingemans** [nbadal/ktlint-intellij-plugin#486]
When changing the ktlint settings, ask the user whether files need to be reformatted:
* none: user will reformat files manually
* open files: only currently opened files are reformatted
* all: al kotlin files in project are reformatted

**Comment by tgkprog** [comment]
Yes please, in general all tools/ AI should have a way to access all files not only open files, when we want a one time scan (could be several times day in case of AI) occasionally for ktlint

**Comment by paul-dingemans** [comment]
> Yes please, in general all tools/ AI should have a way to access all files not only open files, when we want a one time scan (could be several times day in case of AI) occasionally for ktlint

What you are looking for is the action "Format with ktlint" which is already available. This will check all files in the project (including files which are currently not opened in the editor). This issue is about reformatting files which are currently open in the IDEA after changing the settings. The idea (no pun intended) is that opened files immediately should reflect the newly save settings.
